src/plugins/intel_gpu/include/intel_gpu/primitives/bucketize.hpp

```diff
@@ -0,0 +1,42 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+#include "primitive.hpp"
+
+namespace cldnn {
+/// @addtogroup cpp_api C++ API
+/// @{
+/// @addtogroup cpp_topology Network Topology
+/// @{
+/// @addtogroup cpp_primitives Primitives
+/// @{
+
+/// @brief Bucketize operation bucketizes the input based on boundaries.
+struct bucketize : primitive_base<bucketize> {
+    CLDNN_DECLARE_PRIMITIVE(bucketize)
+
+    /// @brief Constructs bucketize primitive.
+    /// @param id This primitive id.
+    /// @param input Input primitive id.
+    /// @param buckets Sorted unique boundaries for buckets primitive id.
+    /// @param output_type Output tensor type.
+    /// @param with_right_bound Indicates whether bucket includes the right or the left edge of interval.
+    bucketize(const primitive_id& id,
+              const primitive_id& input,
+              const primitive_id& buckets,
+              data_types output_type = data_types::i64,
+              bool with_right_bound = true,
+              const primitive_id& ext_prim_id = {},
+              const padding& output_padding = {})
```

Comment:
why do you need this parameter?

Reply:
I'm not sure whether we really it for Bucketize operation, but we have this parameter in almost all other operations and it's used inside a primitive base.

Reply:
So I can leave this?